app/src/main/res/layout/main_activity.xml

```diff
@@ -1,7 +1,129 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/container"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".MainActivity" />
+    android:id="@+id/mainScreen"
+    android:layout_width="fill_parent"
+    android:layout_height="fill_parent"
+    android:background="#0C0B0B"
+    android:backgroundTint="#e00122"
```

Comment:
@romigben These colors should be factored out into the colors/xml resource

Reply:
I've fixed those now, anything with a color is referenced by the colors/xml resource now.